### DISCUSSION TITLE
local vs override naming

### DISCUSSION THREAD

**Discussion by Chemaclass** [gacela-project/gacela#83] [Ideas]
Currently, the default concept name for the last config file that overrides any key-value from the config is known as `local`.
This is actually the default setup for the config:
```php
Gacela::bootstrap($projectRootDir, [
    'config' => [
        'path' => 'config/*.php',
        'path_local' => 'config/local.php',
    ],
]);
```
I am wondering if does it make more sense to use `override` word instead (or similar)? 🤔 
```php
'path_override' => 'config/override.php',
// or maybe keep the config key 'path_local' but change the default config local name to `override.php`?
'path_local' => 'config/override.php',
```
In the end it's not super relevant, and just a matter of naming because the functionality wont be affected.
> You don't like `override`? You can also share other alternatives in this thread.

**Comment by Chemaclass** [comment]
@JesusValera @Tito-Kati what do you think about this? I mean the concept itself, right now it's known as `local`, but I think `override` is way more expressive and realistic.

**Reply by JesusValeraDev** [reply]
I am fine with
```php
Gacela::bootstrap($projectRootDir, [
    'config' => [
        'path' => 'config/*.php',
        'override' => 'config/local.php',
    ],
]);
```

`override` or `override_path` seems better to me, but I am also fine with `path_local` :)

**Reply by Anas-debug** [reply]
Is this the routing part?

**Reply by Chemaclass** [reply]
@JesusValera Actually, now that I think about it again, I think this is not really relevant anymore, because with the new `GacelaConfig` this will be an internal variable naming for the second argument from the `addAppConfig()`. See:
```php
public function addAppConfig(string $path, string $pathLocal = '', $reader = null): self
```

I think after merging https://github.com/gacela-project/gacela/pull/146 we could do a little refactoring renaming `pathLocal` to `overridePath`, for example. And the user wont be affected at all from this.

@Anas-debug "_Is this the routing part?_" this was the old way to altering the config from gacela in the bootstrap (by using an array). This isn't working anymore. The current way is here: https://gacela-project.com/docs/bootstrap/ and the new way that it's coming soon is this one: https://github.com/gacela-project/gacela/pull/146
